docs-source/content/userguide/cicd/how-to-copy-domains.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "How to Copy Domains"
+date: 2019-04-11T13:48:15-04:00
+draft: false
+weight: 5
+description: "How to copy domains."
+---
+
+The recommended approach to save a copy of a domain is to simply zip (or tar) 
+the domain directory.  However there is a very important caveat with this 
+recommendation - when you unzip the domain, it must go back into exactly 
+the same location (Domain Home) in the (new) file system.  Using this 
+approach will maintain the same domain encryption key.  
```

Comment:
In a Docker image the domain home is always the same as long as the domain name is the same.